pal/minigame/wechat.ts

```diff
@@ -19,7 +19,16 @@ minigame.wx.onWheel = wx.onWheel?.bind(wx);
 // #endregion platform related
 
 // #region SystemInfo
-let _cachedSystemInfo: SystemInfo = wx.getSystemInfoSync();
+let _cachedSystemInfo: SystemInfo;
+function updateCachedSystemInfo () {
+    _cachedSystemInfo = wx.getSystemInfoSync();
+    if (_cachedSystemInfo.system.toLowerCase().indexOf('ios') >= 0) {
+        _cachedSystemInfo.platform = 'ios';
+    } else {
+        _cachedSystemInfo.platform = 'android';
+    }
```

Comment:
this removes the `window` platform, which affect the Wechat PC platform

Reply:
This brunch is only for wechat-emoji-project usage.